README.md

```diff
@@ -81,66 +53,11 @@ Once installed, you can run a temporary deployment in dev mode (all data is in-m
 coder server --dev
 ```
 
-Use `coder --help` to get a complete list of flags and environment variables.
-
-## Creating your first template and workspace
-
-In a new terminal window, run the following to copy a sample template:
-
-```bash
-coder templates init
-```
-
-Follow the CLI instructions to modify and create the template specific for your
-usage (e.g., a template to **Develop in Linux on Google Cloud**).
-
-Create a workspace using your template:
-
-```bash
-coder create --template="yourTemplate" <workspaceName>
-```
-
-Connect to your workspace via SSH:
-
-```bash
-coder ssh <workspaceName>
-```
-
-## Modifying templates
-
-You can edit the Terraform template using a sample template:
-
-```sh
-coder templates init
-cd gcp-linux/
-vim main.tf
-coder templates update gcp-linux
-```
+Use `coder --help` to get a complete list of flags and environment variables. For a full walkthrough, follow [this guide](./docs/walkthrough.md).
 
 ## Documentation
 
-- [About Coder](./docs/about.md#about-coder)
-  - [Why remote development](./docs/about.md#why-remote-development)
-  - [Why Coder](./docs/about.md#why-coder)
-  - [What Coder is not](./docs/about.md#what-coder-is-not)
-  - [Comparison: Coder vs. [product]](./docs/about.md#comparison)
-- [Templates](./docs/templates.md)
-  - [Manage templates](./docs/templates.md#manage-templates)
-  - [Persistent and ephemeral
-    resources](./docs/templates.md#persistent-and-ephemeral-resources)
-  - [Parameters](./docs/templates.md#parameters)
-- [Workspaces](./docs/workspaces.md)
-  - [Create workspaces](./docs/workspaces.md#create-workspaces)
-  - [Connect with SSH](./docs/workspaces.md#connect-with-ssh)
-  - [Editors and IDEs](./docs/workspaces.md#editors-and-ides)
-  - [Workspace lifecycle](./docs/workspaces.md#workspace-lifecycle)
-  - [Updating workspaces](./docs/workspaces.md#updating-workspaces)
-
-## Community
-
-Join the community on [Discord](https://discord.gg/coder) and [Twitter](https://twitter.com/coderhq) #coder!
-
-[Suggest improvements and report problems](https://github.com/coder/coder/issues/new/choose)
+Visit our docs [here](./docs/index.md).
```

Comment:
I called it `index.md` because that was what first came to mind. Open to other suggestions.